std/typecons.d

```diff
@@ -797,6 +845,29 @@ template Tuple(Specs...)
             import std.conv : to;
             return this.to!string;
         }
+
+        ///
+        unittest
```

Comment:
Initially it wasn't my idea. I wouldn't do it also, but it's been done a lot in the same file. Should we change it everywhere? It sounds reasonable though it's probably a lot of work. Doesn't the compiler recognize template-independence and opt it out?